Dockerfile.ppmseq

```diff
@@ -0,0 +1,49 @@
+FROM python:3.9-bullseye as build
+
+ENV PYTHONDONTWRITEBYTECODE=1 \
+    PYTHONUNBUFFERED=1 \
+    PYTHONFAULTHANDLER=1 \
+    PATH=/usr/bin/poetry/bin:$PATH \
+    POETRY_VIRTUALENVS_IN_PROJECT=true \
+    POETRY_NO_INTERACTION=1 \
+    PYTHON_KEYRING_BACKEND=keyring.backends.null.Keyring \
+    POETRY_HOME=/usr/bin/poetry
+
+RUN curl -sSL https://install.python-poetry.org > ./install-poetry.py && \
+    python ./install-poetry.py && \
+    rm ./install-poetry.py
+
+# Create virtualenv for deployment
+ENV VIRTUAL_ENV=/opt/venv
+RUN python -m venv $VIRTUAL_ENV
+ENV PATH="$VIRTUAL_ENV/bin:$PATH"
+
+COPY . .
+COPY ./utilities_env/ppmseq/pyproject.toml ./pyproject.toml
```

Comment:
@itamark-ug ?

Reply:
@ilyasoifer I think this might be a good idea (missed the setup folder :-)).
I think we can skip the extra "utilities_env" and just move the ppmseq folder under setup, what do you think?

Reply:
@itamark-ug - sounds lovely